canine/localization/delocalization.py

```diff
@@ -49,12 +54,40 @@ def main(output_dir, jobId, patterns, copy):
                             os.symlink(os.path.abspath(target), dest)
                     else:
                         shutil.copytree(target, dest)
-                manifest.write("{}\t{}\t{}\n".format(
+
+                # compute checksum
+                if name not in {'stdout', 'stderr'}:
+                    try:
+                        subprocess.check_call(
+                          "sha1sum {target} | awk '{{ print $1 }}' > {output}".format(
```

Comment:
How portable is `sha1sum`? Can we rely on that on most linux distros?

Reply:
Good point, I've only tested it on Ubuntu (maximal and minimal). However, ultimately, whether `sha1sum` is portable is moot: we want to switch this to whatever CRC-based hashing scheme `gsutil` uses, so that the hash of a local file will not differ from the hash of a bucket file retrieved using `gsutil`.